augur/routes/contributor_reports.py

```diff
@@ -356,11 +356,14 @@ def get_repo_id_start_date_and_end_date():
 
         now = datetime.datetime.now()
 
-        repo_id = int(request.args.get('repo_id'))
```

Comment:
If 'repo_id' is in the request.args array is None (which means it isn't specified in the query parameters), casting it to an int would result in an error because you cannot cast None to an int. Therefore I moved the cast to after I've checked if the 'repo_id' is specified in the request.args array